src/lib/platform/EiScreen.cpp

```diff
@@ -85,11 +94,25 @@ EiScreen::~EiScreen()
     events_->set_buffer(nullptr);
     events_->remove_handler(EventType::SYSTEM, events_->getSystemTarget());
 
-    ei_device_unref(ei_pointer_);
-    ei_device_unref(ei_keyboard_);
-    ei_device_unref(ei_abs_);
+    if (ei_pointer_) {
+        free(ei_device_get_user_data(ei_pointer_));
+        ei_device_set_user_data(ei_pointer_, NULL);
```

Comment:
Any reason to use  `NULL` over `nullptr` here ?

Reply:
oops, no, just decades of C programming coming through :) good catch, fixed, thanks.